lib/grape/util/stackable_values.rb

```diff
@@ -3,16 +3,16 @@ module Util
     class StackableValues
       attr_accessor :inherited_values
       attr_reader :new_values
-      attr_reader :froozen_values
+      attr_reader :frozen_values
```

Comment:
Those froozen values were just extra frozen. Thanks for spotting this ;)

![](http://media4.giphy.com/media/laypv6pCIEDCM/giphy.gif)